.github/workflows/test.yml

```diff
@@ -161,6 +161,7 @@ jobs:
           path: ./tests/e2e-profile.out
 
   repo-analysis:
+    if: ${{ false }} # Disabled due to requiring SonarCloud integration
```

Comment:
It seems ideal to continue to run the tests above, but this require a `SONAR_TOKEN` for SonarCloud. I think this isn't necessary for us so I'd prefer not to set this up.